connectors/src/connectors/webcrawler/temporal/activities.ts

```diff
@@ -56,11 +58,29 @@ export async function crawlWebsiteByConnectorId(connectorId: ModelId) {
     {
       maxRequestsPerCrawl: MAX_PAGES,
       maxConcurrency: CONCURRENCY,
-
+      maxRequestsPerMinute: 60, // 5 requests per second to avoid overloading the target website
+      requestHandlerTimeoutSecs: REQUEST_HANDLING_TIMEOUT,
       async requestHandler({ $, request, enqueueLinks }) {
         Context.current().heartbeat({
           type: "http_request",
         });
+
+        // try-catch allowing activity cancellation by temporal (timeout, or signal)
+        try {
+          await Context.current().sleep(1);
```

Comment:
Is this second or milliseconds?

Reply:
mili

Reply:
I can switch to sleep(0)
Is the rest of the PR ok?

Reply:
If milli then 1 is fine 👍